GVFS/GVFS/CommandLine/LogVerb.cs

```diff
@@ -60,10 +60,14 @@ public override void Execute()
                 this.DisplayMostRecent(gvfsLogsRoot, GVFSConstants.LogFileTypes.Dehydrate);
                 this.DisplayMostRecent(gvfsLogsRoot, GVFSConstants.LogFileTypes.Repair);
 
-                string serviceLogsRoot = Paths.GetServiceLogsPath(this.ServiceName);
+                string serviceLogsRoot = Path.Combine(
+                    GVFSPlatform.Instance.GetDataRootForGVFSComponent(GVFSConstants.Service.ServiceName),
+                    GVFSConstants.Service.LogDirectory);
                 this.DisplayMostRecent(serviceLogsRoot, GVFSConstants.LogFileTypes.Service);
 
-                string autoUpgradeLogsRoot = Paths.GetServiceLogsPath(ProductUpgraderInfo.UpgradeDirectoryName);
+                string autoUpgradeLogsRoot = Path.Combine(
+                    GVFSPlatform.Instance.GetDataRootForGVFSComponent(ProductUpgraderInfo.GVFSUpgradeComponentName),
+                    ProductUpgraderInfo.LogDirectory);

```

Comment:
Is this the same as `ProductUpgraderInfo.GetLogDirectoryPath()`?

Reply:
Yes.